jingo_minify/tests.py

```diff
@@ -240,7 +243,7 @@ def test_static_override():
 def test_css(getmtime, time):
     getmtime.return_value = 1
     time.return_value = 1
-    env = jingo.env
+    env = jingo.get_env()
```

Comment:
My bad--this updates `requirements.txt` already. We should definitely add a line to `setup.py`, though.